config/constants.py

```diff
@@ -61,6 +63,13 @@
 ETH_MIM_CRV_VAULT = "0x19E4d89e0cB807ea21B8CEF02df5eAA99A110dA5"
 ETH_BADGER_WBTC_CRV_VAULT = "0xeC1c717A3b02582A4Aa2275260C583095536b613"
 
+CRITICAL_VAULTS = [ETH_BVECVX_STRATEGY]
+
+
+class DiscordRoles(Enum):
```

Comment:
we have existing enums file here https://github.com/Badger-Finance/python-keepers/blob/main/config/enums.py

Reply:
Moved it there